components/tabs/stories/tabs.stories.js

```diff
@@ -81,38 +119,156 @@ export default {
 Default.args = {};
 
-// ********* VRT ONLY ********* //
-export const WithForcedColors = TabsGroups.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
-WithForcedColors.parameters = {
-	chromatic: {
-		forcedColors: "active",
-		modes: disableDefaultModes
-	},
-};
-
 // ********* DOCS ONLY ********* //
-export const Vertical = Template.bind({});
-Vertical.tags = ["!dev"];
+/**
+ * Vertical tabs should be used when horizontal space is more generous and when the list of sections is greater than can be presented to the user in a horizontal format. 
+ */
+export const Vertical = VerticalGroup.bind({});
 Vertical.args = {
 	orientation: "vertical",
 };
+Vertical.tags = ["!dev"];
 Vertical.parameters = {
-	chromatic: {
-		disableSnapshot: true,
-	},
+	chromatic: { disableSnapshot: true },
 };
 
-export const Overflow = Template.bind({});
-Overflow.tags = ["!dev"];
+export const VerticalRight = Template.bind({});
+VerticalRight.args = {
+	...Vertical.args, 
+	hasRightAlignedTabs: true,
+};
+VerticalRight.storyName = "Vertical right";
+VerticalRight.tags = ["!dev"];
+VerticalRight.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When there are too many tabs to fit horizontally across the viewport, the tabs component can be displayed as a [quiet picker](/docs/components-picker--docs). The example below is non-functional.
+ * 
+ * When appropriate, alternative methods of overflowing tabs such as horizontal scrolling can be used.
+ */
+export const Overflow = OverflowGroup.bind({});
 Overflow.args = {
 	orientation: "overflow",
 };
+Overflow.tags = ["!dev"];
 Overflow.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.args = {...Default.args};
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Quiet tabs have no visible divider across the tab items apart from the one that shows the selected tab item. These should be used as sub-level navigation or for containers.
+ */
+export const Quiet = QuietGroup.bind({});
+Quiet.args = {
+	isQuiet: true,
+};
+Quiet.tags = ["!dev"];
+Quiet.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Compact tabs should never be used without the quiet variation.
+ */
+export const Compact = CompactGroup.bind({});
+Compact.args = {
+	isCompact: true,
+	isQuiet: true,
+};
+Compact.tags = ["!dev"];
+Compact.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A tab item in a disabled state shows that the tab item exists, but is not available in that circumstance.
+ */
+export const Disabled = Template.bind({});
+Disabled.args = {
+	content: [
+		{
+			id: "tab-1",
```

Comment:
Do we want to let the template.js set a default id for tabs?

Reply:
Let me know if this is what you're thinking. I haven't used `getRandomId` yet, but I looked at some examples. There should be an id now on the tabs component, as well as each tab item. https://github.com/adobe/spectrum-css/pull/2925/commits/bcfe278b0d88e006628d1890d351b9403225ce4d

Reply:
Love it! Added bonus, now there's no risk of a duplicate ID error during testing. 🥳